src/basic/terminal-util.c

```diff
@@ -303,6 +303,19 @@ int reset_terminal_fd(int fd, bool switch_to_text) {
         return r;
 }
 
+int restore_terminal(const char *name, bool reldisp) {
```

Comment:
this call could really use some better naming. It only applies to VTs afaics, hence it should probably be called "vt_something" or so?

Also, a quick comment somewhere about what this actually really does would be appreciated.

Reply:
Indeed, let's call it "vt_release"